jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/TableAssert.java

```diff
@@ -36,13 +36,16 @@ public Compare onlyOne() {
         public String type;
         TableAssert tAssert;
         boolean exact;
-        private Compare(int count, TableAssert tAssert, boolean exact) {
+        String[] messages;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)